### PR TITLE
Add Driver Help For Measat VSAT Broadband Management

### PR DESCRIPTION
**Pull request by jwoSkyline**

### CODE REVIEW

File: connector/doc/Measat_VSAT_Broadband_Management
Comment (MariekeGO) [inline connector/doc/Measat_VSAT_Broadband_Management:1]: This seems to be a duplicate of the other new page you've added, with the same UID. However, this page does not have the .md extension, and some of the content is different. Could you check which of the two pages is correct, and make sure only the correct page is added, with the .md extension?

Comment (jwoSkyline) [reply connector/doc/Measat_VSAT_Broadband_Management:1]: Thank you for noticing that, I must have added my draft page without realizing. I will remove this incorrect page.

File: connector/doc/Measat_VSAT_Broadband_Management.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+To set up the MySQL connection, configure the following parameters on the General page of the element:
+
+- **Database Server**: The IP address of the MySQL server.
+- **Database User Name**: The user name of the account to log into the MySQL server.
+- **Database Password**: The password of the account to log into the MySQL server.
+- **Database Name**: The name of the database being accessed on the MySQL server.
+
+The Site Count archive database queries the state_change_log table to update the net modem statuses.
+
+The Congestion archive database queries the nms_group_qos_changes table to update the Group QOS and congestion data.
```
Comment (MariekeGO) [inline connector/doc/Measat_VSAT_Broadband_Management.md:50]: I'm not sure how to interpret this. Is it the state_change_log table in the Site Count archive database that is queried by the connector to update the net modem statuses, and the nms_group_qos_changes table in the Congestion archive database that is queried to update the Group QOS and congestion data?

Comment (jwoSkyline) [reply connector/doc/Measat_VSAT_Broadband_Management.md:50]: Yes it is. I will rephrase it to make it clearer.

**Review by MariekeGO — APPROVED** [review]
